README.md

```diff
@@ -1 +1,57 @@
-# psc-discrepancy-parser
\ No newline at end of file
+# psc-discrepancy-parser
+This AWS Lambda service processes batches of CSV-format survey results (each survey gathered discrepancies with a company's Person(s) of Significant Control (PSC)), transforming a survey (in the form of a CSV record) into JSON and sending it to CHIPS.
+
+The batches of survey results are sent by email to an address whose emails will automatically be put in an S3 bucket in a particular folder. Files arriving in that folder trigger this Lambda to run. The Lambda will attempt to read the file as an email containing a CSV attachment in a particular format, transforming each CSV record to a PscDiscrepancySurvey JSON object and sending it to CHIPS over an HTTP POST. If any of these steps fail, the file is moved to a failed-to-process folder. If all succeed, the file is moved to a successfully-processed folder.
+
+## Installation
+TODO
+
+## Usage
+There is not much usage to this. Using the Lambda amounts to installing it (see above) and causing a survey to be emailed to the pertinent email address. The Lambda will then be run automatically.
+
+## Support
+### The support process for dealing with a bug
+Until [FAML-180 Make new PSC Discrepancy Contact creation idempotent](https://companieshouse.atlassian.net/browse/FAML-180) is fixed, hold off on re-processing any emails that have, for whatever reason, failed to be processed by the Lambda. Once that JIRA is fixed, dealing with an issue in the Lambda should proceed as follows:
+1. Support should be alerted to an issue in the Lambda because an ERROR log was picked up by the AWS Cloudwatch for the Lambda's logs.
```

Comment:
This suggests the alert will be an automated process but I believe it will be manual at least temporarily, are they more likely to check the S3 `rejected` folder manually to look for issues?

Reply:
Need to talk to support. Bruce believes that they will be monitoring the logs rather than the bucket.

Reply:
Going to create a JIRA to track this.